### ISSUE TITLE
Able to start animations from the beginning now?

### ISSUE THREAD

**Issue by ikrowni** [Aircoookie/WLED#457]
Sorry to open another issue on this but after the new update i saw that the previous was closed but i dont see any mention of the setting being added in. Is this a new feature now or not until later?

**Comment by Aircoookie** [comment]
Hi, I believe I partially managed to add this. Animations will still not start from the beginning if the effect changes, because that would cause significant visual jumps on multi segment and multi controller environments. What I managed to do though is have them start from the beginning if the light is turned on. I hope this will help in your use case :)

**Comment by ikrowni** [comment]
Thanks this should work perfect then.
